website/docs/docs/quickstarts/dbt-core/codespace-qs.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Quickstart for dbt Core using GitHub Codespaces"
+id: codespace
+description: "Quickly get set up to start running dbt commands in a codespace environment."
+sidebar_label: "GitHub Codespace quickstart"
+---
+
+In this quickstart guide, you’ll learn how to create a codespace and be able to execute the `dbt build` command from it in _less than 5 minutes_. 
+
+dbt Labs provides a [GitHub Codespace](https://docs.github.com/en/codespaces/overview) template that you (and anyone else) can reuse to create a complete dbt environment with a working and runnable project. When you create the codespace, the [dev container](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers) creates a fully functioning dbt environment, connects to a DuckDB database, and loads a year of fictional Jaffle Shop data into the database for you. Jaffle Shop is a fictional cafe selling food and beverages in several US cities.
+
+:::tip Check out our Jaffle Shop template 
+The [README](https://github.com/dbt-labs/jaffle-shop-template#readme) for the Jaffle Shop template also provides instructions on how to do this, along with animated GIFs. You can check those out, too.  
+
+:::
+
+## Prerequisites
+
+- To use the dbt CLI, it's important that you know some basics of the terminal. In particular, you should understand `cd`, `ls` , and `pwd` to navigate through the directory structure of your computer easily.
+- You have a [GitHub account](https://github.com/join).
+
+## Create a codespace
+
+1. Go to the `jaffle-shop-template` [repository](https://github.com/dbt-labs/jaffle-shop-template) after you log in to your GitHub account. 
+1. Click **Use this template** at the top of the page and choose **Create new repository**.
+1. Click **Create repository from template** when you’re done setting the options for your new repository.
+1. Click **Code** (at the top of the new repository’s page). Under the **Codespaces** tab,  choose **Create codespace on main**. Depending on how you've configured your computer's setting, this either opens a new browser tab with VSCode running in it or opens a new VSCode window with the codespace in it. 
+1. Wait for the codespace to finish building by waiting for the `postCreateCommand` command to complete; this can take several minutes:
+
+    <Lightbox src="/img/codespace-quickstart/postCreateCommand.png" title="Wait for postCreateCommand to complete" />
+
+    When this command completes, you can start using the codespace development environment. The terminal the command ran in will close and you will get a prompt in a brand new terminal. 
+
```

Comment:
For all of these instructions below will they follow the exact same workflows regardless of whether they're working from the browser or the app with command line? Are there limitations on the Codespace browser tools they need to be aware of?

For example, will `dbt docs serve` execute the same if they're using the browser Codespace without the data on their physical drive?